packages/react-error-overlay/build.js

```diff
@@ -5,11 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 const webpack = require('webpack');
-const chalk = require('chalk');
+const Chalk = require('chalk');
```

Comment:
This is the script that *builds the error overlay itself*. It's not the error overlay.

It is not necessary to change this here, and as you can see on the screenshot, it doesn't fix the problem.

Reply:
tnx, i'm working on it...
@gaearon